std/array.d

```diff
@@ -263,8 +263,9 @@ unittest
 }
 
 /**
-Returns a newly allocated associative array out of elements of the input range,
-which must be a range of tuples (Key, Value).
+Params: r = An input range of tuples of keys and values.
```

Comment:
Quick question: Will the docs be generated properly with a lack of a blank line between `Params` and `Returns` (and `Returns and `See Also`) sections?